openapi-spec/swagger.yaml

```diff
@@ -2044,8 +2497,8 @@ parameters:
     required: true
     description: The UUID of the volume attachment.
     type: string
-  snapshotId:
-    name: snapshotId
+  shareSnapshotId:
```

Comment:
snapshotId is volume snapshot id, you need to add another variable.

Reply:
Done